assets/dev/scss/editor/preview/preview.scss

```diff
@@ -59,6 +92,41 @@ html.elementor-html {
 		}
 	}
 
+	.e-grid-outline {
+		display: none;
+	}
+
+	.elementor-element-editable.e-grid {
+
+		&.e-con-full {
+
+			> .e-grid-outline {
+
+				@include e-grid-outline-full-width;
+			}
+		}
+
+		&.e-con-boxed {
+
+			> .e-con-inner{
+
+				position: relative;
+
+				> .e-grid-outline {
+
+					@include e-grid-outline-boxed;
+				}
+			}
+		}
+
+		.e-grid-outline-item {
+			border: 1px dashed #{$e-gray-300};
```

Comment:
Could you double check if there is a css variable for this? I know that Ariel made some changes to the editor colours.

Reply:
Good comment!
I replaced it with the following`scss` variable: `$editor-light`